src/AzureDeprecation.APIs.REST/Contracts/AzureCloud.cs

```diff
@@ -0,0 +1,11 @@
+namespace AzureDeprecation.APIs.REST.Contracts;
```

Comment:
Is there a reason why this is duplicated and not re-used from the contracts project?

Reply:
With that copy it would be much easily to change response if frontend require something specific.
I think it's bad idea to have only one type used on all levels of the application